standard/conversions.md

```diff
@@ -71,20 +71,48 @@ The pre-defined implicit conversions always succeed and never cause exceptions t
 
 > *Note*: Properly designed user-defined implicit conversions should exhibit these characteristics as well. *end note*
 
-For the purposes of conversion, the types `object` and `dynamic` are *runtime interchangeable* (§10.2.2).
+For the purposes of conversion, the types `object` and `dynamic` are identity convertible (§10.2.2).
 
 However, dynamic conversions ([§10.2.10](conversions.md#10210-implicit-dynamic-conversions) and [§10.3.8](conversions.md#1038-explicit-dynamic-conversions)) apply only to expressions of type `dynamic` ([§8.2.4](types.md#824-the-dynamic-type)).
 
 ### 10.2.2 Identity conversion
 
-An identity conversion converts from any type to the same type or a type that is equivalent at runtime. One reason this conversion exists is so that a type `T` or an expression of type `T` can be said to be convertible to `T` itself.
-
-Two types are *identity convertible* when an identity conversion exists between two types. The following such identity conversions exist:
+An identity conversion converts from any type to the same type or a type that is equivalent at runtime. One reason this conversion exists is so that a type `T` or an expression of type `T` can be said to be convertible to `T` itself. The following identity conversions exist:
 
+- Between `T` and `T`, for any type `T`.
 - Between `object` and `dynamic`.
 - Between all tuple types with the same arity, and the corresponding constructed `ValueTuple<...>` type, when an identity conversion exists between each pair of corresponding element types.
 - Between types constructed from the same generic type where there exists an identity conversion between each corresponding type argument.
```

Comment:
Should "same" still be here? I suspect it's okay because there are no (open) generic types with identity conversions, but I'm just checking. (I can imagine the potential in the future for this to change to "types constructed from identity convertible generic types where [...]")

Reply:
Should I add "...constructed from the same **open** generic type where ..." (emphasis only in comment, not meant for a spec change).